docs/interfaces/tensorboard.rst

```diff
@@ -208,23 +194,16 @@ hosted on agent machines but they do not occupy GPUs.
 Can I log additional TensorBoard events beyond what Determined logs automatically?
 ==================================================================================
 
-Yes; any additional TFEvent files that are written to ``/tmp/tensorboard`` inside a trial container
-will be accessible via TensorBoard. For example, to log a custom TensorBoard event using PyTorch:
-
-.. code:: python
-
-   from torch.utils.tensorboard import SummaryWriter
-
-   writer = SummaryWriter(log_dir="/tmp/tensorboard")
-   writer.add_scalar("my_metric", np.random.random(), batch_idx)
-
-For more details, as well as examples of how to do this with TF Estimator and TF Keras models, refer
-to the :ref:`TensorBoard How-To Guide <data-in-tensorboard>`.
+Yes; any additional TFEvent files that are written to the appropriate path during training will be
+accessible to TensorBoard. The appropriate path varies by worker rank, and can be obtained via one
+of the following functions:
```

Comment:
Yes, any additional TFEvent files written to the appropriate path during training are accessible to TensorBoard. The appropriate path varies by worker rank and can be obtained via one of the following functions: